src/main/java/net/snowflake/client/jdbc/SnowflakeDriver.java

```diff
@@ -26,6 +31,8 @@
  * loading
  */
 public class SnowflakeDriver implements Driver {
+  private static final SFLogger logger = SFLoggerFactory.getLogger(SnowflakeDriver.class);
+  protected static final String AUTO_CONNECTION_STRING_PREFIX = "jdbc:snowflake:auto";
```

Comment:
maybe we want to keep it public? not protected, nor internal?

Reply:
Yes we can